generators/app/index.js

```diff
@@ -115,11 +116,9 @@ module.exports = JhipsterGenerator.extend({
                 if (err) {
                     this.warning('Node is not found on your computer.');
                 } else {
-                    var nodeVersionMajor = stdout.split('.')[0].replace(/v/g, '');
-                    var nodeVersionMinor = stdout.split('.')[1];
-                    var nodeVersion = stdout.replace(/\n/g, '');
-                    if (nodeVersionMajor < 6 || (nodeVersionMajor >= 6 && nodeVersionMinor < 9)) {
-                        this.warning('Your node version is too old (' + nodeVersion + '). You should use at least Node ' + chalk.bold('v6.9.0+'));
+                    var nodeVersion = semver.clean(stdout);
+                    if (semver.lt(nodeVersion, '6.9.0')) {
```

Comment:
can we put the supported version in the constants and use that here and in the package.json templates please? so that we dont have to change in multiple places